admin/src/templates/users.html

```diff
@@ -25,15 +25,15 @@
         <div class="col-xs-1"></div>
       </div>
 
-      <div class="row selection" ng-repeat="user in users" ng-click="editUser(user)">
+      <div class="row selection" ng-repeat="user in users" ng-click="editUser(user)" ng-class="{'inactive': user.inactive}">
```

Comment:
I think this means it'll _look_ disabled but clicking on the row will pop up the modal. Then if you submit that modal I'm guessing you'll get an error because the _user doc is deleted.

I think you could modify this either by returning early in the controller `editUser` function, or something like...

```suggestion
      <div class="row selection" ng-repeat="user in users" ng-click="!user.inactive && editUser(user)" ng-class="{'inactive': user.inactive}">
```

Another solution would be to show the modal but disable the inputs and submit button so you can see more info about the user but not update the info. I don't think this is necessary for now.

Reply:
😱 I didn't noticed this one, thanks!

Reply:
Fixed.